examples/new_model.py

```diff
@@ -42,16 +42,10 @@ def initialize(self, u, z):
             }
     
     def dx(self, x, u):
-        # apply_process_noise is used to add process noise to each step
-        return self.apply_process_noise({
-            'x': x['v'],
-            'v': self.parameters['g'] # Acceleration of gravity
-        })
+        return x
```

Comment:
Here you're returning the state (x, v) instead of the first derivative of state (v, a | a=g).

You should just be removing the call to apply_process_noise

Reply:
so I should just return self instead of returning self.apply_process_noise()?